javascript/2-functions/hw-2/scripts.js

```diff
@@ -1 +1,14 @@
 // Your JS code is here
+function Num(idOne,idTwo) {
+  if (idOne > idTwo) {
+    return (
+      idOne
+    )
+  } else {
+    return (
+      idTwo
+    )
+  }
+}
```

Comment:
this function could be improved. 
1. is there any specific reason for you to use those parenthesis after return on line 4 , 6, 8, 10 ? if not, those could be removed.
2. Also, else could be removed in order for this function to look more clear.